Dockerfile.rosbag

```
@@ -0,0 +1,48 @@
+FROM ros:galactic-ros1-bridge
+ENV DEBIAN_FRONTEND noninteractive
+
+# ROS
+ENV ROS1_DISTRO noetic
+ENV ROS2_DISTRO galactic
+
+# ROS2
+RUN set -x && \
+  sh -c 'echo "deb http://packages.ros.org/ros/ubuntu $(lsb_release -sc) main" > /etc/apt/sources.list.d/ros-latest.list' && \
+  apt-get update -y -qq && \
+  : "install ROS1 and ROS2" && \
+  apt-get install -y -qq \
+    ros-${ROS2_DISTRO}-image-transport \
+    ros-${ROS2_DISTRO}-cv-bridge \
+    ros-${ROS2_DISTRO}-image-tools \
+    ros-foxy-rosbag2-bag-v2-plugins \
+    ros-foxy-ros2bag \
+    ros-foxy-rosbag2-transport \
+    ros-foxy-rosbag2-converter-default-plugins \
+    ros-foxy-rosbag2-storage-default-plugins \
```

Do not mix up foxy and galactic.

```suggestion
    ros-${ROS2_DISTRO}-rosbag2-bag-v2-plugins \
    ros-${ROS2_DISTRO}-ros2bag \
    ros-${ROS2_DISTRO}-rosbag2-transport \
    ros-${ROS2_DISTRO}-rosbag2-converter-default-plugins \
    ros-${ROS2_DISTRO}-rosbag2-storage-default-plugins \
```

---

@ymd-stella 
Thank you so much for your review comments. After updating all changes as suggested by you, I got bellow error while docker build.
`E: Unable to locate package ros-galactic-rosbag2-converter-default-plugins`

So just as a work around i replaced   `${ROS2_DISTRO}` with `foxy` in `ros-${ROS2_DISTRO}-rosbag2-converter-default-plugins`. This time build was successful. But i was unable to play rosbag. Got below error.

```
Failed to load entry point 'play': /opt/ros/galactic/lib/librosbag2_storage.so: undefined symbol: _ZNK12class_loader11ClassLoader15isLibraryLoadedEv
Traceback (most recent call last):
  File "/opt/ros/galactic/bin/ros2", line 11, in <module>
    load_entry_point('ros2cli==0.13.1', 'console_scripts', 'ros2')()
  File "/opt/ros/galactic/lib/python3.8/site-packages/ros2cli/cli.py", line 39, in main
    add_subparsers_on_demand(
  File "/opt/ros/galactic/lib/python3.8/site-packages/ros2cli/command/__init__.py", line 250, in add_subparsers_on_demand
    extension.add_arguments(
  File "/opt/ros/galactic/lib/python3.8/site-packages/ros2bag/command/bag.py", line 26, in add_arguments
    add_subparsers_on_demand(
  File "/opt/ros/galactic/lib/python3.8/site-packages/ros2cli/command/__init__.py", line 237, in add_subparsers_on_demand
    extension = command_extensions[name]
KeyError: 'play'
```
Commands followed are as below:
root@hostname:/ros2_ws#source /opt/ros/${ROS1_DISTRO}/setup.bash
root@hostname:/ros2_ws#source /opt/ros/foxy/setup.bash
root@hostname:/ros2_ws#ros2 bag play -s rosbag_v2 /path/to/rosbagfile.bag

---

Rather than replacing it with a version of foxy, simply remove it.

---

Updated as per review comment.